codeserver-python/rockcraft.yaml

```
@@ -0,0 +1,246 @@
+#Based on the following Dockerfiles:
+# https://github.com/kubeflow/kubeflow/blob/v1.10.0-rc.1/components/example-notebook-servers/base/Dockerfile
+# https://github.com/kubeflow/kubeflow/blob/v1.10.0-rc.1/components/example-notebook-servers/codeserver/Dockerfile
+# https://github.com/kubeflow/kubeflow/blob/v1.10.0-rc.1/components/example-notebook-servers/codeserver-python/Dockerfile
+name: codeserver-python
+summary: An image for using Codeserver & Jupyter on CPUs
+description: |
+  This image is used as part of the Charmed Kubeflow product. It is deployed
+  in a user namespace, and enables users to perform CPU-accelerated tasks
+  using PyTorch from within a Jupyter Notebook.
+
+  Both PyTorch and Jupyter are installed in Conda environment, which is
+  automatically activated.
+version: "1.10.0-rc.1"
```

suggestion(blocking): Let's update the tag
```suggestion
version: "1.10.0"
```

---

Done